scss/index.scss

```diff
@@ -70,6 +70,23 @@ $g-gutter-half: ($grid-gutter-width / 2) + px;
 	}
 }
 
+// .g--gutter--<breakpoints>
+// Generates gutter settings for each defined breakpoint for better gutter control in responsive designs
+
+@each $name, $value in $breakpoints {
```

Comment:
Can you include the generated CSS in the commit as well, as that’ll need to be updated.  Depending on the amount of extra CSS, it might be worth having this as an option in the `variables.scss` like the responsive grid settings – could call it `$responsive-gutters` and have it as false by default so that it can be switched on as an option when pulled in.

Also will need to bump the package,json to `1.2.0`.

Otherwise, looks good 👍